azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/chat/components/MessageListView.kt

```diff
@@ -89,6 +89,25 @@ private fun requestPages(
     }
 }
 
+@Composable
+private fun sendReadReceipt(
+    scrollState: LazyListState,
+    messages: List<MessageViewModel>,
+    dispatch: Dispatch
+) {
+    val firstVisibleItemIndex = scrollState.firstVisibleItemIndex
+    val currentBottomMessage = messages[messages.count() - firstVisibleItemIndex - 1]
```

Comment:
Have you checked the case when there is no message in the repo.

Reply:
thanks for appointing this problem.